cylc/flow/etc/examples/cycle-over-irregular-dates/inter-dependent/flow.cylc

```diff
@@ -0,0 +1,51 @@
+#!Jinja2
+
+{%
+    set DATES = [
+        '2000-01-01T00:00Z',
+        '2000-01-01T06:00Z',
+        '2000-03-05T12:00Z',
+        '2000-05-28T13:36Z',
+        '2001-01-05T23:24',
+        '2002-04-30T04:20',
+    ]
+%}
+
+[meta]
+    title = Irregular cycling example
+    description = """
+        A workflow that runs a group of tasks on arbitrary dates
+        with inter-cycle dependencies between those dates.
+    """
+
+[scheduling]
+    initial cycle point = 2000
+    [[graph]]
+        # define the tasks you want to run on startup
+        R1 = install
+
+        # run this graph every year
+        P1Y = """
+            install[^] => prep
+        """
```

Comment:
I'm not sure what this item illustrates in this example.

Reply:
Not all irregular cycling workflows are going to need this, but some will, so I added some text explaining the `strftime` and `loop.first` thinggies to help users with more advanced problems.